vsphere/resource_vsphere_host.go

```diff
@@ -104,6 +112,40 @@ func resourceVsphereHost() *schema.Resource {
 				Default:      "disabled",
 				ValidateFunc: validation.StringInSlice([]string{"disabled", "normal", "strict"}, true),
 			},
+			"services": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"ntpd": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"enabled": {
+										Type:        schema.TypeBool,
+										Optional:    true,
```

Comment:
Are `enabled` and `policy` really optional?
If not I suggest that you mark them as required and leave the containing `ntpd` block as optional to prevent users from making easy mistakes

Reply:
updated to false, need to test to see what it does

Reply:
i have updated the code, i also added the stop of the service if requested.

Reply:
so i completed testing, i tested add host to vCenter, then did changes to each setting to verify all changes will be done